packages/plugin-core/src/test/suite-integ/WindowDecorations.test.ts

```diff
@@ -0,0 +1,85 @@
+import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import { describe } from "mocha";
+import { NoteUtils } from "@dendronhq/common-all";
+import * as vscode from "vscode";
+import { VSCodeUtils } from "../../utils";
+import { expect } from "../testUtilsv2";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+import { updateDecorations } from "../../features/windowDecorations";
+import _ from "lodash";
+
+/** Check if the ranges decorated by `decorations` contains `text` */
+function assertTextDecorated(
+  text: string,
+  decorations: vscode.DecorationOptions[],
+  document: vscode.TextDocument
+) {
+  for (const decoration of decorations) {
+    if (document.getText(decoration.range) === text) return;
```

Comment:
nitpick. can we use the expect(...).toEqual(...) syntax?
this way, we get proper reporting on number of tests run